pylinac/core/profile.py

```diff
@@ -159,6 +160,31 @@ def values(self, value):
             raise TypeError("Values must be a numpy array")
         self._values = value.astype(float)
 
+    @property
+    def dpmm(self) -> NumberLike:
```

Comment:
This is a pretty good idea as we can convert to physical distance easier inherently.